lib/src/core/utils/prefs_utils.dart

```diff
@@ -10,6 +10,30 @@ class PrefsUtils {
 
   static SharedPreferences get prefs => getIt<SharedPreferences>();
 
+  static bool get isRemember => prefs.getBool("isRemember") ?? false;
+
+  static Future setIsRemember(bool value) async =>
+      await prefs.setBool("isRemember", value);
+
+  static String? get email => prefs.getString("email");
+
+  static String? get password => prefs.getString("password");
+
+  static Future setRemember({
+    required String email,
+    required String password,
+  }) async {
+    await setIsRemember(true);
```

Comment:
replace to await Future.wait([ request ])

Reply:
I fixed it